### PR TITLE
Speed up build by combining signing requests

### PR DESCRIPTION
**Pull request by rohit21agrawal**
This combines 3 separate sign requests for vsix, build tools vsix  and nupkgs into one request.

### CODE REVIEW

**Review by dtivel — APPROVED** [review]

File: build/sign.proj [hunk 1]
```diff
@@ -13,8 +13,8 @@
     </CommonMSBuildProperties>
     <OutDir>$(RepositoryRootDirectory)</OutDir>
     <SignTargetsDependOn>BatchSign</SignTargetsDependOn>
-    <SignTargetsDependOn Condition="'$(SignVsix)' == 'true'">GetOutputVsix</SignTargetsDependOn>
-    <SignTargetsDependOn Condition="'$(SignNupkgs)' == 'true'">GetOutputNupkgs</SignTargetsDependOn>
+    <SignTargetsDependOn Condition="'$(SignPackages)' == 'true'">GetOutputNupkgs</SignTargetsDependOn>
+    <SignTargetsDependOn Condition="'$(SignPackages)' == 'true' AND '$(BuildRTM)' != 'true'">GetOutputNupkgs;GetOutputVsix</SignTargetsDependOn>
```
Comment (dtivel) [inline build/sign.proj:17]: Why this change?

    AND '$(BuildRTM)' != 'true'

Comment (rohit21agrawal) [reply build/sign.proj:17]: BuildRTM is a multiplier that runs two build in parallel for values true and false. When true, it produces nupkgs without the build number. In this build, the vsix is not produced, and hence the condition.

**Review by nkolev92 — APPROVED** [review]
